Tiltfile

```diff
@@ -113,6 +113,28 @@ COPY --from=tilt-helper /usr/bin/kubectl /usr/bin/kubectl
     },
 }
 
+addon_settings_files = []
+
+# Create a data structure to hold information about addons.
+addons = {
+    "test-extension": {
+        "context": "./test/extension",
+        "image": "gcr.io/k8s-staging-cluster-api/test-extension",
+        "container_name": "extension",
+        "binary_name": "extender",
+        "live_reload_deps": ["main.go", "handlers"],
+        "label": "test-extension",
+        "resource_deps": ["capi_controller"],
+        "kustomize_substitutions": {
```

Comment:
I probably just missed it, but where is this field used?

Reply:
This is now refactored and left over - addons should now define kustomize-substitutions in the core tilt-settings file.